python/packages/nisar/products/writers/BaseL2WriterSingleInput.py

```diff
@@ -42,6 +42,52 @@
 LUT_1D_RG_DATASETS_LUT_EXPANSION_ALONG_AZ_MARGIN_IN_PIXELS = 5
 
 
+def get_static_layers_data_access(static_layers_data_access_runconfig,
+                                  granule_id):
+    """
+    Read static layers data access template from the input runconfig field
+    `static_layers_data_access` and replace placeholder(s) with
+    the actual values.
```

Comment:
Is this description still accurate...? (It's not.) Please update per the new standalone function.